src/bundle/ui-dev/src/modules/universal-discovery/components/finder/finder.branch.js

```diff
@@ -130,10 +142,6 @@ const FinderBranch = ({ locationData, itemsPerPage }) => {
         dispatchLoadedLocationsAction({ type: 'UPDATE_LOCATIONS', data });
     }, [loadedLocations, dispatchLoadedLocationsAction, isLoading]);
 
-    if (!subitems.length && !collapsed) {
-        return null;
```

Comment:
As far as I remember this check prevents displaying an empty branch when it is the last item in the tree.